laa_court_data_api_app/internal/court_data_adaptor_client.py

```diff
@@ -56,8 +57,14 @@ async def __send_request(self, method: str, endpoint: str,
         if token is None:
             return token
 
+        return await self.__call_endpoint(oauth_client, token, method, endpoint, params, headers, body)
+
+    @circuit()
```

Comment:
A few questions just to help me understand.
Are you happy with the default config of 5 attempts, 30 sec recovery timeout and catch all exceptions?
Will this work with you catching the exception and returning None or does it depend on returning the exception?

Reply:
Using the default config seems reasonable - we are only wanting to retry on calls being thrown and not status code returns. 5 attempts over a 30 second period should prevent any overload and if this doesnt work still we continue as normal. The decorator just calls the inner function and returns the exception after exceeding time/attempts so no extra work is required in that sense